core/src/main/resources/lib/layout/breadcrumbBar_es.properties

```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2020, CloudBees, Inc., Romen Rodriguez-Gil
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+dashboard=Panel de control
```

Comment:
Should this be title-cased? It is elsewhere:

https://github.com/jenkinsci/jenkins/blob/22aa2e6e766074d11249893e3f35e0b99e20d3d0/core/src/main/resources/hudson/logging/LogRecorderManager/sidepanel_es.properties#L23

Reply:
I would keep it Dashboard TBH, Panel de Control is too long I believe

Reply:
I had the same doubt, @fqueiruga, but I saw it was being used already in other places and since I was trying to test the new breadcrumb at least in another language, I added it.

I'll use title case per Daniel's suggestion, but no hard opinion if it is decided to remove it.